Octokit.Tests.Integration/Reactive/ObservableRepositoriesClientTests.cs

```diff
@@ -27,5 +28,26 @@ public class TheGetMethod
                 Assert.False(repository2.Fork);
             }
         }
+
+        public class TheGetAllPublicSinceMethod
+        {
+            [IntegrationTest]
```

Comment:
Currently, yes it most likely will :smile: but when I created the test, it ran for a few seconds, so I can at least say I tested the feature :grimacing: